tensorboard/webapp/plugins/npmi/views/annotations_list/annotation/annotation_container.ts

```diff
@@ -83,6 +86,21 @@ export class AnnotationContainer {
         };
       })
     );
+  readonly experimentIds$ = this.store.pipe(select(getExperimentIdsFromRoute));
+  readonly runMaps$ = this.experimentIds$.pipe(
+    filter((experimentIds) => Boolean(experimentIds)),
+    switchMap((experimentIds) => {
+      return combineLatest(
+        experimentIds!.map((experimentId) => {
+          return this.store.select(selectors.getRuns, {experimentId});
+        })
+      );
+    }),
+    map((runMaps) => {
+      let maps: Run[] = [];
+      return maps.concat(...runMaps);
```

Comment:
there is `Array.prototype.flat` you can use. Also, this is an array, not a map (object/dict/Map).